internal/driver/mobile/device.go

```diff
@@ -45,14 +47,14 @@ func (*device) HasKeyboard() bool {
 	return false
 }
 
-func (*device) ShowVirtualKeyboard() {
-	showVirtualKeyboard(mobile.DefaultKeyboard)
+func (dev *device) ShowVirtualKeyboard() {
```

Comment:
Isn't single letter names for polymer receivers the standard/ideomatic way in Go?

Reply:
I was just matching what was there. I can tidy that up on another PR - there is a conflict internally to do that refactor which I will get to the bottom of.